mappings/net/minecraft/container/CombinationContainer.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4861 net/minecraft/container/CombinationContainer
+	FIELD field_22479 resultInv Lnet/minecraft/class_1263;
+	FIELD field_22480 inputsInv Lnet/minecraft/class_1263;
+	FIELD field_22481 blockContext Lnet/minecraft/class_3914;
+	FIELD field_22482 player Lnet/minecraft/class_1657;
+	METHOD <init> (Lnet/minecraft/class_3917;ILnet/minecraft/class_1661;Lnet/minecraft/class_3914;)V
+		ARG 1 type
+		ARG 2 syncId
+	METHOD method_24923 onCombine (Lnet/minecraft/class_1657;Lnet/minecraft/class_1799;)Lnet/minecraft/class_1799;
+		ARG 1 combiner
+		ARG 2 result
+	METHOD method_24925 isContainerBlock (Lnet/minecraft/class_2680;)Z
+	METHOD method_24927 canTakeResult (Lnet/minecraft/class_1657;Z)Z
+		ARG 2 hasStack
```

Comment:
This needs a better name that doesn't have the `has` prefix (as per [naming conventions](https://github.com/FabricMC/yarn/blob/1b7aa91a4ddc43fd259bc4011834e7e5112143b8/CONVENTIONS.md))

Reply:
```suggestion
		ARG 2 nonEmpty
```

Reply:
`hasStack` is fine IMO

Reply:
I like present personally